src/Fantomas.Core.Tests/KeepIndentInBranchTests.fs

```diff
@@ -2183,3 +2183,55 @@ module Foo =
 
         failwith ""
 """
+
+[<Test>]
+let ``if keyword and elseBranch align, 2973`` () =
+    formatSourceString
+        false
+        """
+module Program =
+    let main _ =
+        if false then 1 else
+        printfn "hi!"
+        0
+"""
+        config
+    |> prepend newline
+    |> should
+        equal
+        """
+module Program =
+    let main _ =
+        if false then
+            1
+        else
+
+        printfn "hi!"
+        0
+"""
+
+[<Test>]
+let ``elif keyword and elseBranch align`` () =
```

Comment:
For completeness, would you mind adding a test for the following too?

```fsharp
async {
    if not (proc.Start ()) then return Error "failed to start" else
    use stdout = proc.StandardOutput
    let! ct = Async.CancellationToken
    return! Async.AwaitTask (stdout.ReadToEnd ct)
}
```

Reply:
Sure thing!